docs/book/v2/quick-start.md

```diff
@@ -10,20 +10,25 @@ For example, to register an `AlbumListHandler` located in the `module/Applicatio
 your `Application` module, add the following route to `module/Application/config/module.config.php`:
 
 ```php
-use Application\Handler\AlbumListHandler;
+use Application\Handler\BlogListHandler;
 use Laminas\Mvc\Middleware\PipeSpec;
 use Laminas\Router\Http\Literal;
 
 return [
+    'service_manager' => [
+        'factories' => [
+            Middleware\BlogListMiddleware::class => Factory\BlogListMiddlewareFactory::class,
+        ],
+    ],
```

Comment:
This is the important part of this pull request, because it fixed the problem. However, it should be added as a separate code example with its own headline.

Everything else should be dealt with in another proposal.

Reply:
I can recreate the PR in separate parts, if it helps, @voltan / @froschdesign.